src/couch_replicator_api_wrap.erl

```diff
@@ -83,9 +83,16 @@ db_open(#httpdb{} = Db1, _Options, Create) ->
                     ok
                 end)
         end,
-        send_req(Db, [{method, head}],
-            fun(200, _, _) ->
-                {ok, Db};
+        send_req(Db, [{method, get}],
+            fun(200, _, {Props}) ->
+                case get_value(<<"update_seq">>, Props) of
```

Comment:
instance_start_time is always 0 in couchdb 2.0 (because of the multiple nodes involved, if we used the actual start times we'd end up restarting replication continuously).